nearcache_map.go

```diff
@@ -62,8 +84,52 @@ func newNearCacheMap(nc *inearcache.NearCache, ncc *nearcache.Config, ss *serial
 	return ncm, nil
 }
 
-func (ncm *nearCacheMap) registerInvalidationListener() {
-	fmt.Println("IMPLEMENT ME: registerInvalidationListener")
+func (ncm *nearCacheMap) Destroy(ctx context.Context, name string) error {
+	ncm.lg.Trace(func() string {
+		return fmt.Sprintf("hazelcast.nearCacheMap.Destroy: %s", name)
+	})
+	s := ncm.invalidationListenerID.Load()
+	if s == nil {
+		return nil
+	}
+	// removeNearCacheInvalidationListener
```

Comment:
removeNearCacheInvalidationListener method in Java has these lines as well so it may be reasonable to move the comment:
```suggestion
	// removeNearCacheInvalidationListener
	s := ncm.invalidationListenerID.Load()
	if s == nil {
		return nil
	}
```

Reply:
https://github.com/hazelcast/hazelcast-go-client/pull/807/commits/be6741a1d8290e83a043566cce08ef14ea4e0ed3